Source/Samples/CalloutSample/CalloutSample.Droid/CalloutSample.Droid.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\packages\Xamarin.Build.Download.0.4.11\build\Xamarin.Build.Download.props" Condition="Exists('..\..\..\packages\Xamarin.Build.Download.0.4.11\build\Xamarin.Build.Download.props')" />
```

Comment:
was this upgrade necessary to solve the null issue?

Reply:
ok man, i will rollback to old package! but i think package too old, we should update them 👍